guides/common/modules/proc_configuring-logging-to-journal.adoc

```diff
@@ -23,3 +23,14 @@ endif::[]
 --foreman-logging-type journald \
 --foreman-proxy-log JOURNAL
 ----
++
+. Optional: To reset `journald` logging back to default:
```

Comment:
This isn't an optional step as part of configuring journald logging but rather a procedure to configure file based logging. Wouldn't it be better to separate this out to its own procedure?

I'd also not refer to this as reset journald logging but rather reconfigure Foreman and Foreman Proxy to use default (file based) logging.

Reply:
@ekohl Both methods of logging are now described as separate procedures. I had to rewrite the entire file to do it. Can you please take another look?